src/components/V2Tabs/V2Tab.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render, fireEvent } from 'react-testing-library';
+
+import { V2Tab } from '.';
+import { oltStyles } from '../..';
+
+describe('Notifications Render Correctly', () => {
+  test('renders active Tab', () => {
+    const { getByText } = render(
+      <V2Tab active label="tab label" value="tab1" />,
+    );
+    getByText('tab label');
+    const tab = getByText('tab label');
+    expect(tab.classList.contains(oltStyles.V2TabsLink)).toBeTruthy();
+    expect(tab.classList.contains(oltStyles.isActive)).toBeTruthy();
```

Comment:
```suggestion
    expect(tab.classList.contains(oltStyles.isActive)).toBe(true);
```
https://vincenttunru.com/toBeTruthy-vs-toBe-true/